### ISSUE TITLE
Please choose an open source license for this website

### ISSUE THREAD

**Issue by justwheel** [reginaelyse/reginapizza.github.io#1]
Hi @reginapizza, I saw you on Twitter and you have a really cool website design. :smile: I wanted to see how you did things and see if I could improve my website HTML/CSS based on how you did it on your website.

However when you make a project on GitHub and there is no LICENSE, by default it falls under default copyright law in the United States. This is really restrictive and doesn't allow other folks to (legally) attribute credit to you and reuse/change parts of your code in other websites.

I suggest using Choose a License, a site maintained by GitHub, to pick an open source license. This site explains how licenses work in plain English and tells you what people can or cannot do with your original code. All licenses require that you receive attribution from anyone else that takes your work and iterates on it!

https://choosealicense.com/

If you used a license like the MIT License or 3-Clause BSD License, it would make it easier for me to borrow your awesome ideas and CSS legally, and attribute credit back to you publicly. Of course, pick a license that makes the most sense to you. :)

Btw, congrats on your internship with RH OpenShift team! :smile:

**Comment by reginaelyse** [comment]
Hello,

I actually did pay for a license to use this website, as you can see the
credit given at the very bottom of the webpage, but I am no expert in
copyright so I could have entirely done something wrong with crediting it
on github. I think it was only $13 when I paid for the license and you can
change it in any way you want. The majority of changes that I made to the
original template was mostly just deleting parts I didn't want. If I wanted
to I could make something similar but it would certainly take me a lot of
time; when I was putting it together I was in the final stages of my
bootcamp and had hardly any time to design and code my own portfolio, which
is why I opted to buy the license for the template.

I would feel like I would be discrediting the hard work of the creator if I
said you could use my files on github, but technically there's nothing
stopping you. I also do not feel that it is entirely my own work since I
mostly just deleted things from the original template, so I do not feel the
need to create a license for my version of it. I would encourage you to
check out the original creator and his work here
<http://shreethemes.in/cristino/>. There is a lot of cool stuff that he
created that I didn't end up using, but that you may want.

If you are struggling to get HTML/CSS working to your liking I wouldn't
mind taking a look at it if you link a github repo.

Thank you for the congratulations, I sincerely appreciate it! Good luck on
your own portfolio endeavors!
